src/System.Private.ServiceModel/tests/Common/Scenarios/ScenarioTestHelpers.cs

```diff
@@ -1,18 +1,24 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using Infrastructure.Common;
 using System;
```

Comment:
Also the coding standards say when the top level of the namespace is different, there should be a blank line between them.